### DISCUSSION TITLE
RPi Pico 2 W

### DISCUSSION THREAD

**Discussion by DeeJay** [gusmanb/logicanalyzer#176] [Ideas]
Something to add to the 'wishlist'.

Our Project lead and developer is probably already aware, but just to flag up that Raspberry Pi released the wireless-enabled variant of their RP2350-based Raspberry Pi Pico_2_W board towards the end of 2024. [Raspberry Pi Pico 2 W on sale now at $7](https://www.raspberrypi.com/news/raspberry-pi-pico-2-w-on-sale-now/)

**Comment by gusmanb** [comment]
I already have one on the way, once I get it will check the implementation :)

**Comment by 41south** [comment]
Hi, have just received my V6 board (thansk very much!) and wondering if you've confirmed that the Pico 2 W is supported?

**Reply by gusmanb** [reply]
Sorry, nope, didn't had enough time, I'm on a business trip and will  still take a couple of weeks to be back at home.

**Comment by gusmanb** [comment]
There is an experimental build available with support for the Pico 2 W.

https://mega.nz/folder/SGxDHAZL#afLGgQbJAaqOYXjhJBwokQ

Feel free to test it and provide feedback.

Remember to upgrade both, software and firmware :)

**Reply by JonathanLindsey** [reply]
Is the code for this available?

Version 6.0.0.0 releases contain variants for the following boards
```
PICO
PICO_2
PICO_2_Turbo
PICO_Turbo
PICO_W
PICO_W_WIFI
ZERO
ZERO_Turbo
```

But not any for the Pico 2 W. Lots of variants there. :-P

I looked to see if the Pico 2 W option is in the master branch but it doesn't appear to be.
https://github.com/gusmanb/logicanalyzer/blob/39a7a0b43fa2de03927cc17ce397b2fc4fbe314b/Firmware/LogicAnalyzer_V2/LogicAnalyzer_Build_Settings.cmake#L2

**Comment by DeeJay** [comment]
Noting that there is now a beta release of v6.5 that includes support for RPi Pico 2 W.

And there is also an update from Raspberry PI about a new revision of their RP235x microcontroller chip.

[https://www.raspberrypi.com/news/rp2350-a4-rp2354-and-a-new-hacking-challenge/](https://www.raspberrypi.com/news/rp2350-a4-rp2354-and-a-new-hacking-challenge/)

**Reply by DeeJay** [reply]
To further note that Raspberry Pi have also released v2.2.0 of the Pico SDK.

[https://github.com/raspberrypi/pico-sdk/releases/tag/2.2.0](https://github.com/raspberrypi/pico-sdk/releases/tag/2.2.0)

**Comment by kd7eir** [comment]
Are there any concerns with the relocated Debug connections on the Pico 2 W with the assembled board?
